cfgov/ask_cfpb/models/pages.py

```diff
@@ -33,6 +35,32 @@
 ENGLISH_DISCLAIMER_SNIPPET_TITLE = 'Legal disclaimer for consumer materials'
 SPANISH_DISCLAIMER_SNIPPET_TITLE = (
     'Legal disclaimer for consumer materials (in Spanish)')
+CONSUMER_TOOLS_PORTAL_PAGES = {
```

Comment:
@higs4281 I agree, using a hard-coded list is definitely suboptimal. It was a quick way to get the breadcrumbs set up for the portal page release, though. I'm still trying to find a better long-term solution that won't require updating when pages are added or changed.